pkgs/applications/version-management/gitoxide/default.nix

```diff
@@ -26,10 +31,17 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ cmake pkg-config ];
 
-  buildInputs = [ curl ] ++ (if stdenv.isDarwin
+  buildInputs = [ curl installShellFiles ] ++ (if stdenv.isDarwin
```

Comment:
Could you please move `installShellFiles` to `nativeBuildInputs`? 
It is not a run-time dependency but a build-time dependency.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/289330/commits/b9c5c429f6d7169e48cf287cefd1afec57f38b18